intrastat_product/report/intrastat_product_report_xls.py

```diff
@@ -53,6 +65,17 @@ def _get_template(self, declaration):
                 },
                 "width": 14,
             },
+            "src_dest_country_code": {
+                "header": {
+                    "type": "string",
+                    "value": self._("Country of Origin/Destination"),
+                },
+                "line": {
+                    "type": "string",
+                    "value": self._render("line.src_dest_country_id.code"),
+                },
+                "width": 28,
```

Comment:
please change to:
line.src_dest_country_code

In case of Northern Ireland the src_dest_country_code will be XI whereas the Country Code will be GB.